[2025-02-08T09:18:02.624+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T09:18:02.642+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.load_data scheduled__2025-02-07T00:00:00+00:00 [queued]>
[2025-02-08T09:18:02.650+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.load_data scheduled__2025-02-07T00:00:00+00:00 [queued]>
[2025-02-08T09:18:02.650+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-08T09:18:02.662+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_data> on 2025-02-07 00:00:00+00:00
[2025-02-08T09:18:02.666+0000] {standard_task_runner.py:72} INFO - Started process 95 to run task
[2025-02-08T09:18:02.668+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'load_data', 'scheduled__2025-02-07T00:00:00+00:00', '--job-id', '172', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp_tmsbha1']
[2025-02-08T09:18:02.671+0000] {standard_task_runner.py:105} INFO - Job 172: Subtask load_data
[2025-02-08T09:18:02.689+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-08T09:18:02.732+0000] {task_command.py:467} INFO - Running <TaskInstance: first_dag.load_data scheduled__2025-02-07T00:00:00+00:00 [running]> on host f4d9e66dfb95
[2025-02-08T09:18:02.733+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-08T09:18:02.811+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='first_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-07T00:00:00+00:00'
[2025-02-08T09:18:02.812+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T09:18:03.189+0000] {base.py:84} INFO - Retrieving connection 'my_dwh_conn'
[2025-02-08T09:18:03.982+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T09:18:03.992+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T09:18:03.993+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=load_data, run_id=scheduled__2025-02-07T00:00:00+00:00, execution_date=20250207T000000, start_date=20250208T091802, end_date=20250208T091803
[2025-02-08T09:18:04.045+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T09:18:04.061+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-08T09:18:04.074+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
